src/main/java/com/maddyhome/idea/vim/newapi/IjVimCaret.kt

```diff
@@ -145,10 +145,10 @@ class IjVimCaret(val caret: Caret) : VimCaretBase() {
   override fun hashCode(): Int = this.caret.hashCode()
 }
 
-val VimCaret.ij: Caret
+inline val VimCaret.ij: Caret
```

Comment:
Just curious, any advantages from making this field inlined?

Reply:
Possibly not :) I didn't measure anything, but it felt like an overhead to call a function that simply does a cast and property access, especially when we've got that same pattern repeated in the code. It just means there's no difference to putting the pattern in the calling code or using this helper function, which is a lot nicer.

I've always wondered if it's worth putting `IjVimEditor` into `Editor`s user data, to prevent allocating too frequently, too.